scv-prototype-frontend/src/app/App.jsx

```diff
@@ -33,16 +33,18 @@ const App = (props) => {
 				<Router>
 					<MainLayout>
 						<Switch>
-							<Route path="/" component={Home} exact />
-							<Route path="/benefits-services" component={BenefitsServices} exact />
-							<Route path="/book-appointment" component={BookAppointment} exact />
-							<Route path="/inbox" component={Inbox} exact />
-							<Route path="/jobs-skills" component={JobsSkills} exact />
-							<Route path="/lets-connect" component={LetsConnect} exact />
-							<Route path="/notifications" component={Notifications} exact />
-							<Route path="/preferences" component={Preferences} exact />
+							<Route exact path="/" >
+								<Redirect to="/home" />
```

Comment:
I don't love this redirect.. why not just use `/` as home? This is pretty typical, I think.